lib/TokenManager.ts

```diff
@@ -303,6 +405,10 @@ export class TokenManager {
       options.storage = 'cookie';
     }
 
+    if (isIE11OrLess()) {
+      options._storageEventDelay = 1000;
```

Comment:
this should be a **default** if running on IE11, it should not override the value we have set